lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,14 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zone">
+                <% _.each(preOptions, function(time_zone) { %>
+                   <option value="<%- time_zone.time_zone %>"><%- time_zone.description %></option>
+                <% }); %>
+                <option disabled>──────────</option>
```

Comment:
@andy-armstrong how should I go about this divider line to make sure it's accessible as well? There's a picture of it at the top of the PR. This line is my temporary solution.